src/components/balance/ModalTransferAmount.vue

```diff
@@ -16,18 +16,20 @@
             <h3
               class="text-lg font-extrabold text-blue-900 dark:text-white mb-6 text-center"
             >
-              Transfer XXX
+              Transfer {{ unitToken }}
             </h3>
 
             <button
               type="button"
               class="w-full bg-blue-500 dark:bg-blue-800 text-white rounded-lg px-5 py-5 mb-4 relative hover:bg-blue-600 dark:hover:bg-blue-700 focus:outline-none focus:ring focus:ring-blue-100 dark:focus:ring-blue-400"
             >
               <span class="block text-left font-bold text-sm mb-2"
-                >XXX Balance</span
+                >{{ unitToken }} Balance</span
```

Comment:
The dropdown button doesn't seem to do anything here? Can you make it into a non-interactable balance display or add some sort of behavior to it?
<img width="610" alt="Screen Shot 2021-04-26 at 7 35 26 PM" src="https://user-images.githubusercontent.com/40356749/116070882-17544200-a6c8-11eb-9f62-d8996849587d.png">

Reply:
yes, I think we can remove the dropdown button, because it's useless. what's the meaning of a non-interactable balance display?

Reply:
It just means removing the dropdown button, sorry for the confusing word choices